platforms/sdl/main.cpp

```diff
@@ -1,4 +1,4 @@
-#include <cstdarg>
+#include <stdarg.h>
```

Comment:
Why?

Reply:
cstdarg does not work on C++03, stdarg.h does